pyflagser/flagser.py

```diff
@@ -41,6 +41,16 @@ def flagser(flag_matrix, min_dimension=0, max_dimension=np.inf, directed=True,
         often ``100,000``. Increase for higher precision, decrease for faster
         computation. A negative value computes highest possible precision.
 
+    filtration : string, optional, default: `"max"`
```

Comment:
Still not right. I've fixed it myself as it is difficult to explain here :)

Reply:
Thank you and sorry, not quite familiar with sphinx